dev/integration_tests/display_cutout_rotation/test_driver/display_cutout_test_test.dart

```diff
@@ -8,22 +8,29 @@ import 'dart:io';
 
 import 'package:flutter_driver/flutter_driver.dart';
 
-// display_cutout needs a custom driver becuase cutout manipulations needs to be
+// display_cutout needs a custom driver because cutout manipulations needs to be
 // done to a device/emulator in order for the tests to pass.
 Future<void> main() async {
   if (!(Platform.isLinux || Platform.isMacOS)) {
     // Not a fundemental limitation, developer shortcut.
     print('This test must be run on a POSIX host. Skipping...');
     return;
   }
-  final bool adbExists = Process.runSync('which', <String>['adb']).exitCode == 0;
-  if (!adbExists) {
-    print(r'This test needs ADB to exist on the $PATH.');
-    exitCode = 1;
-    return;
+  String adbExecutable = 'adb';
```

Comment:
Did you figure out the ADB issue? Can you remove this debugging code

Reply:
This is not really debugging code at this point. It is code that allows for local and ci based development.

Reply:
it looks like the only thing you're using it for is an API level check? Seems like that could be pulled into the script.

Reply:
+1, it does not make sense to have this here if we're submitting.